doc/lxc.container.conf.sgml.in

```diff
@@ -284,6 +284,25 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
       </variablelist>
     </refsect2>
 
+    <refsect2>
+      <title>Init working directory</title>
+      <para>
+        Sets the absolute path inside the container as the working directory for the containers.
```

Comment:
Specify clearly that lxc will switch to this directory before executing init.

Reply:
fixed